includes/admin/settings/class-settings-advanced.php

```diff
@@ -153,6 +153,31 @@ public function get_settings() {
 								'disabled' => __( 'Disabled', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Setup Page', 'give' ),
+							/* translators: %s: about page URL */
+							'desc'    => sprintf(
+								wp_kses(
+									__( 'Disable this option if you would like to disable the <a href="%s" target="_blank">GiveWP Setup page</a> that displays when GiveWP is first isntalled.', 'give' ),
+									[
+										'a' => [
+											'href'   => [],
+											'target' => [],
+										],
+									]
+								),
+								esc_url( admin_url( 'edit.php?post_type=give_forms&page=give-setup' ) )
+							),
+							'id'      => 'setup_page_enabled',
+							'type'    => 'radio_inline',
+							'default' => give_is_setting_enabled( \Give\Onboarding\Onboarding::isSetupPageEnabled() )
+								? \Give\Onboarding\Onboarding::ENABLED
+								: \Give\Onboarding\Onboarding::DISABLED,
+							'options' => [
+								\Give\Onboarding\Onboarding::ENABLED => __( 'Enabled', 'give' ),
```

Comment:
@kakshak did you set up the development environment correctly because if you did then this must be auto fix by phpcs

Reply:
The git commit hooks are linting the files.